sdk/database/helper/connutil/sql.go

```diff
@@ -34,8 +34,9 @@ const (
 )
 
 const (
-	dbTypePostgres   = "pgx"
-	cloudSQLPostgres = "cloudsql-postgres"
+	dbTypePostgres         = "pgx"
+	dbTypeCloudSQLPostgres = "cloudsql-postgres"
+	dbTypeSnowflake        = "snowflake"
```

Comment:
maybe these aren't the same parameter, but should we universalize the usage between this one and `snowflakeSQLTypeName` in `vault-plugin-database-snowflake` at some point?